.github/workflows/setup-test-workspace/action.yml

```diff
@@ -1,12 +1,17 @@
 name: Setup test workspace
+inputs:
+  ref-slice-ref:
+    description: 'Reference of the ref_slice to checkout'
+    required: false
+    default: ''
```

Comment:
I'm confused by this. If the input isn't set (because it isn't required) and the default is empty-string, what will `actions/checkout@v3` do? Will that be the right thing to do?